text/0000-bionic-mixins.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Defining Mixins for io.buildpacks.stacks.bionic
+- Start Date: 2019-12-05
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+The Platform API specification allows stack maintainers to define stack mixins for a given stack.
+The Cloud Native Buildpacks project informally defines stack build and run images with ID `io.buildpacks.stacks.bionic` as containing exactly the set of Ubuntu packages in the `ubuntu:bionic` image on Docker Hub.
+This RFC proposes that we formalize the existing definition of `io.buildpacks.stacks.bionic` and define mixins on that stack to be Ubuntu 18.04 LTS packages. 
+
+In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Binoic."
```

Comment:
I don't understand this requirement: equivalency seems to be key. For example, say I'm authoring a buildpack for  small niche language.  I *want* to be able to assert that my buildpack should work on anything claiming to be equivalent to Bionic.  I don't see the use of asserting that it will only work for `ubuntu:bionic`.

```suggestion
In general, a stack is a contract that should be more like "Ubuntu Bionic" and less like "Platform XYZ's Re-distribution of Ubuntu Bionic."
```